ansible/testbed-cli.sh

```diff
@@ -240,6 +241,15 @@ function remove_topo
 {
   topology=$1
   passwd=$2
+  remove_keysight_api_server=0
+
+  if [ -n "$3" ]; then
```

Comment:
what if it isn't the 3rd arg? In the call below in L267 we are also passing $@. Better to check if 'remove_keysight_api_server' is present in the arg list and if present , we set remove_api_server to 1 else 0. We can also remove the if condition in L262 and always pass 'remove_api_server' with the appropriate value in the ansible-playbook call

Reply:
The required changes are implemented. Also changes from older commit have been refactored.